pkg/appStore/deployment/service/AppStoreDeploymentService.go

```diff
@@ -445,40 +455,19 @@ func (impl AppStoreDeploymentServiceImpl) InstallApp(installAppVersionRequest *a
 	if err != nil {
 		return nil, err
 	}
-	err = tx.Commit()
-
-	err = impl.installAppPostDbOperation(installAppVersionRequest)
-	if err != nil {
-		return nil, err
+	if !impl.deploymentTypeConfig.HelmInstallAsyncMode {
```

Comment:
this will skip status update for apps deployed via argocd too. Is this expected?